dGame/dComponents/ControllablePhysicsComponent.cpp

```diff
@@ -388,3 +388,29 @@ void ControllablePhysicsComponent::SetStunImmunity(
 		bImmuneToStunUseItem
 	);
 }
+
+void ControllablePhysicsComponent::AddFallSpeed(float value) {
+	m_ActiveFallSpeeds.push_back(value);
+	m_FallSpeed = value;
+	SetGravityScale(value);
```

Comment:
If possible please implement proper dirtying for Gravity scale.  Right now it is not properly dirtied